app/src/main/java/com/sanaa/novix/di/details_modules/remoteDetailsDataSource.kt

```diff
@@ -15,8 +17,14 @@ import retrofit2.Retrofit
 val remoteDetailsDataSource = module {
     singleOf(::RemoteActorDataSourceImpl) bind RemoteActorDataSource::class
     singleOf(::RemoteMovieDataSourceImp) bind RemoteMovieDataSource::class
```

Comment:
why you ate l in RemoteMovieDataSourceImp 
```suggestion
    singleOf(::RemoteMovieDataSourceImpl) bind RemoteMovieDataSource::class
```